### PR TITLE
UIORGS-131 Update table layouts in edit forms to remove redundant headings

### PR DESCRIPTION
**Pull request by NikitaSedyx**
purpose: https://issues.folio.org/browse/UIORGS-131

approach: new property to display fields heading only one + styles (SS is attached)

doc has been updated
test has been added

ss

### CODE REVIEW

**Comment by JohnC-80** [comment]
There needs to be some sort of accessibility note with this that the headings will need id's and the fields will need corresponding `aria-labelledby` attributes applied if this technique is used. Also, if this is to be a prevalent pattern to follow, we should include an example in documentation that includes the accessibility implementation.

**Comment by NikitaSedyx** [comment]
@JohnC-80 IMO accessibility is part of fields and labels definition, I did it with `aria-label`, it's just a heading - not a part of this component

agreed, I'll add an example for this case

**Comment by zburke** [comment]
Do you have a before and after screenshot? Sorry; it isn't clear to me from the ticket what the problem is (there is no "expected results, actual results" comparison), nor from this PR how that problem is addressed. 

Based on @JohnC-80's comment, I am concerned that this makes managing accessibility correctly a manual process that is the responsibility of the developer. Certainly it is possible for a developer to do correctly, but one of the major benefits of using components is to be certain that it is already handled correctly and automatically.

**Comment by NikitaSedyx** [comment]
@zburke we have it now - repeatable labels, and our PO and UX team requested to avoid them
ss

as for accessibility, in this component it's controlled manually by developer because everything is passed in props
if you know how to make it better please advice

**Comment by NikitaSedyx** [comment]
@JohnC-80 I've got you incorrectly, yep, I'll update readme with note, thanks!

File: lib/RepeatableField/RepeatableField.js [hunk 1]
```diff
@@ -82,7 +95,11 @@ class RepeatableField extends Component {
                 </div>
                 {
                   typeof onRemove === 'function' && (
-                    <div className={css.repeatableFieldRemoveItem}>
+                    <div
+                      className={
+                        `${css.repeatableFieldRemoveItem} ${headLabels && css.repeatableFieldRemoveUnlabeledItem}`
```
Comment (ViktorSoroka) [inline lib/RepeatableField/RepeatableField.js]: What would be if `headLabels` in `headLabels && css.repeatableFieldRemoveUnlabeledItem` is falsy? I wonder whether something is set in that case to the class value. If so, I would recommend proceeding with the ternary option.

 `headLabels ? css.repeatableFieldRemoveUnlabeledItem : ''`

Comment (NikitaSedyx) [reply lib/RepeatableField/RepeatableField.js]: actually nothing critical, but yep, it's better

**Comment by NikitaSedyx** [comment]
@JohnC-80 please take a look, thanks

**Comment by NikitaSedyx** [comment]
@zburke and it's react final form

**Pull request merged**